nft-rfc-006.md

```diff
@@ -0,0 +1,860 @@
+properties. Each IID \"namespace\"---to use the CAIP term---defines its
+own Interchain compatible DID method, e.g., did:erc721, did:cosmos,
+did:iov, did:etc. DID Methods that support IID functionality define
+themselves as an IID Method. IID semantics, shared among different IID
+Methods, enable cross-chain interoperability with other IIDs for NFT and
+blockchain functionality.
+
+In short, IIDs **ARE** DIDs.
+
+This makes IIDs compatible with URLs and URIs and widespread tooling of
+the World Wide Web as well as emerging DID technology. It also ensures
+that IIDs are drop-in replacements for RDF subjects, predicates,
+objects, and contexts, allowing RDF statements to use IIDs for
+assertions (as is done with Verifiable Credentials) with exceptional
+rigor.
+
+**Second**, differentiate IIDs as on-chain identifiers with their own
+semantic namespace for references and resources.
+
+A**ll** IIDs refer to on-chain assets. Dereference an IID, you get a
+representation of that asset. Then, each IID can be combined with a path
+part or fragment part to refer to off-chain resources or on-chain
+defined references. Use path parts for dereferencable resources\--those
+that can be retrieved online. Use fragments for pure identifiers defined
+within the IID Document.
+
+In this way, every IID defines its own namespace, within which rigorous
+statements can be made about real-world objects (references) and which
+can be directly downloaded via web technology (resources).
+
+For example, **did:title:abc** might represent a token that is the title
+to a real property, and **did:title:abc\#parcel1** is used to refer to
+that property in attestations associated with that token, such as
+"**did:title:abc\#parcel1"** is located at 123 Main Street, Anytown,
+France. Then, you can use **did:title:abc/photo1.png** to point to an
+image of the property.
+
+By IID convention, if it is desirable to refer to a IID Resource in an
+RDF statement, an IID reference is constructed using the same value as a
+fragment as the path part of the reference. For example, to make an RDF
+statement about the photo at **did:title:abc/photo1.png**, the IID
+Document simply defines an IID Reference of
+**did:title:abc\#photo1.png**.
+
+This distinction avoids the dreaded HTTPRange14 problem because the raw
+IID is always only used to refer to the asset, which is always
+dereferenceable, while IID references are only ever URIs (which are not
+themselves inherently dereferenceable) and IID Resources
+
+Both base IIDs, IID References (with fragments), and IID Resources (with
+path parts), are fully conformant URIs and suitable for use in RDF
+statements.
+
+IIDs resolve to IID documents which contain the minimum required
+metadata for using the NFT. Like a DID Document, it defines a range of
+features for interacting with the subject. It may also provides IID
+specific properties:
+
+**AssociatedResources** -- provides a privacy-enabled way to attach
```

Comment:
Change to Linked Resources (a preferred term which reinforces the concept that these identifiers and metadata can form a graph of verifiable information)